RadarSDK/RadarAPIClient.m

```diff
@@ -138,6 +138,18 @@ - (void)flushReplays:(NSArray<NSDictionary *> *_Nonnull)replays
                            logPayload:NO
                       extendedTimeout:YES
                     completionHandler:^(RadarStatus status, NSDictionary *_Nullable res) {
+                            id eventsObj = res[@"events"];
+
+                            NSArray<RadarEvent *> *events = [RadarEvent eventsFromObject:eventsObj];
+                            if (events.count) {
+                                NSArray *eventsArr = (NSArray *)eventsObj;
+                                NSDictionary *lastEvent = [eventsArr lastObject];
+                                NSDictionary *userObj = lastEvent[@"user"];
+
+                                RadarUser *user = [[RadarUser alloc] initWithObject:userObj];
+                                [[RadarDelegateHolder sharedInstance] didReceiveEvents:events user:user];
```

Comment:
want to sanity check here if this handles if user is undefined / not passed down. would be good to have the option of just passing through events

Reply:
In the transition from 3.1 to 3.2, we made this optional: https://github.com/radarlabs/radar-sdk-ios/blob/559948728bcdb78c937e885a2cf33d8eb46836c1/MIGRATION.md#L67

You can see an example in the trip functions, e.g. https://github.com/radarlabs/radar-sdk-ios/blob/559948728bcdb78c937e885a2cf33d8eb46836c1/RadarSDK/RadarAPIClient.m#L619